topic04-git-introduction/book-git-tutorial/05.Forking.md

```diff
@@ -0,0 +1,38 @@
+# Forking on GitHub
+
+## What is forking?
+
+To fork a repository means to get your own version/copy of the repository, and to be able to edit it without affecting the original repo (repository). 
+
+## How to fork
+
+. You first go onto someone’s github profile
+. Click on one of their repositories 
+. Then click the fork button which is usually located below your profile picture dropdown (see image)
+
+ ![](https://github.com/feedhenry/student-help-guide-content/tree/master/topic04-git-introduction/book-git-tutorial/img/fork.png) 
```

Comment:
Not sure if this image will render using http link. You might need to use relative paths